src/sst/elements/osseous/README.md

```diff
@@ -0,0 +1,72 @@
+#### Osseous
+
+Osseous is a generic RTL Component template, a major portion of ERAS framework, which is updated based on the user requirement to generate dynamically loadable component with SST. The automation flow of the ERAS framework set up by the parser takes in the generic RTL template and C-model header as input and emits final dynamically loadable SST component library integrable with SST.
+
+The repository contain _osseous_ and an example of dynamically loadable component formed out of osseous. 
+Currently, the example is a simple vector shift register(_vecshiftreg_).
+
+```
+osseous
+osseous/examples/vecshiftreg
+```
+To build and register _osseous_ & _vecshiftreg_ and run the shift register example, below are the steps.
+```
+clone the repository containing osseous
+
```

Comment:
This is fine for the test files but there shouldn't be any compilation requirements tied to SST_WORKDIR (see Makefile.am)

Reply:
Replaced SST_WORKDIR to top_srcdir in commit: 475de88 after rebase.